docs/index.rst

```diff
@@ -1,31 +1,70 @@
-=======================
-Scylla Care-Pet Example
-=======================
+.. title:: Scylla Care-Pet Example
 
-.. toctree::
-   :maxdepth: 2
-   :hidden:
+.. raw:: html
 
-   Getting Started with CarePet: A sample IoT App <getting_started>
-   Design and Data Model <design_and_data_model>
-   Care-Pet GitHub Repository <https://github.com/scylladb/care-pet>
-   Care-Pet Blog <https://www.scylladb.com/2020/09/09/carepet-an-example-iot-use-case-for-hands-on-app-developers/>
+  <div class="landing landing--floating">
+
+.. hero-box::
+  :title: Scylla Care-Pet Example
+  :button_icon: fa fa-github
+  :button_url: https://github.com/scylladb/care-pet
+  :button_text: Care-Pet Project
+
+  This example project demonstrates a generic IoT use case for ScyllaDB.
+
+.. raw:: html
+
+  <div class="landing__content landing__content">
+
+.. raw:: html
 
+  <div class="topics-grid grid-container full">
 
-This example project demonstrates a generic IoT use case for ScyllaDB.
+  <div class="grid-x grid-margin-x">
 
-The application allows tracking of pets health indicators and consist of 3 parts:
+.. topic-box::
+  :title: Getting Started with CarePet
+  :link: getting_started
+  :icon: scylla-icon scylla-icon--getting-started
+  :class: large-6
+  :anchor: Get Started
 
-* a collar that reads and pushes sensors data
-* a web app for reading and analyzing the pets' data
-* a database migration tool
+  Create an IoT app from scratch and configure it to use ScyllaDB as the backend datastore.
 
-Pick a topic to get started:
+.. topic-box::
+  :title: Design and Data Model
+  :link: design_and_data_model
+  :icon: scylla-icon scylla-icon--overview
+  :class: large-6
+  :anchor: Learn more
 
-* `Getting Started with CarePet: A sample IoT App <getting_started>`_
+  Learn more about the queries used in the example, make the primary key and clustering key selection, and create the database schema.
 
-* `Design and Data Model <design_and_data_model>`_
+.. topic-box::
+  :title: Care-Pet GitHub Repository
+  :link: https://github.com/scylladb/care-pet
+  :icon: fa fa-github
+  :class: large-6,small-12
+  :anchor: Explore the code
+
+  See the source code, or better yet, contribute to it! (Link opens a new window)
+
+.. topic-box::
+  :title: Care-Pet Blog
+  :icon: fa fa-newspaper
+  :link: https://www.scylladb.com/2020/09/09/carepet-an-example-iot-use-case-for-hands-on-app-developers/
+  :class: large-6,small-12
+  :anchor: Read the article
+
+  Read our blog! (Link opens a new window)
```

Comment:
Do you think it's necessary to add "(Link opens a new window)"?

Reply:
we removed the External Links heading any other suggestions on how to tell them it is an external link?

Reply:
If it's external link, we could add the next icon:

![image](https://user-images.githubusercontent.com/9107969/135267511-5182a031-2b16-443b-a61c-3ae2d1aeeb1e.png)


I'm tracking the improvement here: https://github.com/scylladb/sphinx-scylladb-theme/issues/216